packages/core/src/Asset/cip67.ts

```diff
@@ -1,4 +1,4 @@
-import * as Cardano from '../Cardano';
+import { AssetName } from '../Cardano';
```

Comment:
Even better, as importing from index might still result in circular imports
```suggestion
import { AssetName } from '../Cardano/X';
```

Reply:
6d080359ace should solve this